packages/slice-machine/src/legacy/components/Navigation/SideNavEnvironmentSelector/SideNavEnvironmentSelector.tsx

```diff
@@ -138,6 +139,8 @@ export const SideNavEnvironmentSelector: FC<SideNavEnvironmentSelectorProps> = (
                 }
               />
             ) : undefined}
+
+            {variant === "default" && <LogoutButton />}
```

Comment:
❓ Did you test the use case where we have multiple environments?

Reply:
I did! It works as intended, the screenshot in the description is using a repo with multiple envs.